src/Storages/ObjectStorageQueue/ObjectStorageQueueIFileMetadata.cpp

```diff
@@ -309,32 +320,30 @@ bool ObjectStorageQueueIFileMetadata::trySetProcessing()
 std::optional<ObjectStorageQueueIFileMetadata::SetProcessingResponseIndexes>
 ObjectStorageQueueIFileMetadata::prepareSetProcessingRequests(Coordination::Requests & requests, const std::string & processing_id)
 {
-    if (metadata_ref_count.load() > 1)
```

Comment:
Removed this condition, previously we reused metadata state and took a per file `processing_lock` only in case there were more than one s3queue table on the same server which is following the same keeper path. But now it is possible that a single s3queue table starts several `streamToViews` threads (in case of `parallel_insert = 1`), so we'd better always go into this branch, this fixes a race on local state update.